app/presenters/setup_presenter.rb

```diff
@@ -17,6 +18,11 @@ def steps_visible?
     ).sign_up_progress_visible?
   end
 
+  def remember_device_box_checked?
+    return true if user_opted_remember_device_cookie.nil?
+    ActiveModel::Type::Boolean.new.cast(user_opted_remember_device_cookie)
```

Comment:
TIL this exists! Thank you!!! I've done so many other things to boolean cast in the past 🙈